BootloaderCorePkg/Library/SmbiosInitLib/SmbiosInitLib.c

```diff
@@ -160,6 +172,8 @@ FinalizeSmbios (
   EFI_STATUS                    Status;
   SMBIOS_TABLE_ENTRY_POINT      *SmbiosEntry;
 
+  Status = BuildProcessorInfo ();
```

Comment:
Can we move this to SmbiosInit (), if we can get the Mp info correctly by then?

Reply:
Yes, let me do that.